tests/integrations/test_crossed_integrations.py

```diff
@@ -186,3 +197,96 @@ def validate_kafka_spans(self, producer_interface, consumer_interface, topic):
 
         # returns both span for any custom check
         return producer_span, consumer_span
+
+
+IRRELEVANT_LIBRARY_LOGIC = context.library == "cpp" or context.library == "php"
+
+
+MISSING_LIBRARY_LOGIC = (
+    context.library == "dotnet"
+    or (context.library == "python" and context.weblog_variant != "flask-poc")
+    or (context.library == "golang" and context.weblog_variant != "net-http")
+    or (context.library == "java" and context.weblog_variant != "spring-boot")
+    or (context.library == "nodejs" and context.weblog_variant != "express4")
+    or (context.library == "ruby" and context.weblog_variant != "rails70")
+)
+
+
+@irrelevant(IRRELEVANT_LIBRARY_LOGIC)
+@missing_feature(MISSING_LIBRARY_LOGIC, reason="Weblog endpoint has not been implemented")
+@scenarios.crossed_tracing_libraries
+@coverage.basic
+@features.kafkaspan_creationcontext_propagation_with_dd_trace_js
+class Test_NodeJSKafka(_Test_Kafka):
+    buddy_interface = interfaces.nodejs_buddy
+    buddy = _nodejs_buddy
+    WEBLOG_TO_BUDDY_TOPIC = "Test_NodeJSKafka_weblog_to_buddy"
+    BUDDY_TO_WEBLOG_TOPIC = f"Test_NodeJSKafka_buddy_to_weblog"
```

Comment:
```suggestion
    BUDDY_TO_WEBLOG_TOPIC = "Test_NodeJSKafka_buddy_to_weblog"
```

And same for 10-ish other lines below

Reply:
Done